pkg/model/openstackmodel/servergroup.go

```diff
@@ -129,6 +132,7 @@ func (b *ServerGroupModelBuilder) buildInstances(c *fi.ModelBuilderContext, sg *
 			Role:             fi.String(string(ig.Spec.Role)),
 			Port:             portTask,
 			Metadata:         igMeta,
+			SecurityGroups:   ig.Spec.AdditionalSecurityGroups,
```

Comment:
I do not understand what is purpose of this change because we are not going to change ports in instances? We are not doing that in kops and these ports/instances are managed by kops - that is why I see that this should not be here

Reply:
Well, it's fine to just have it on the port. I added it to the instance because it was possible and compared to other clouds (e.g. AWS) it is the way it is done there also the same way. So technically there is nothing wrong in adding it to the instance, but it will be never used, at least not for managed resources from kops.